src/servers/Http1Server.cc

```diff
@@ -72,6 +72,8 @@ Http::One::Server::noteMoreBodySpaceAvailable(BodyPipe::Pointer)
 Http::Stream *
 Http::One::Server::parseOneRequest()
 {
+    TimeScope parseContext(requestParseStart);
```

Comment:
We cannot easily simplify here (getting rid of TimeScope wrap) because requestParseStart may be used (incorrectly) if we buildFakeRequest() somewhere further (within the same clientParseRequests() iteration). In buildFakeRequest() `request_first_read` should be 'now'.